lib/pages/chat/events/sending_image_info_widget.dart

```diff
@@ -119,4 +128,9 @@ class SendingImageInfoWidget extends StatelessWidget {
       ),
     );
   }
+
+  void _cancelSending() {
+    mediaCancelToken?.cancel();
+    event.remove();
```

Comment:
the return type should be Future<void>, what if event.remove() failed, i see it have return type Future<bool>, you should check the condition and handle error as well